db/coalescing_iterator.cc

```diff
@@ -0,0 +1,46 @@
+//  Copyright (c) Meta Platforms, Inc. and affiliates.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+
+#include "db/coalescing_iterator.h"
+
+#include "db/wide/wide_columns_helper.h"
+
+namespace ROCKSDB_NAMESPACE {
+
+void CoalescingIterator::Coalesce(const WideColumns& columns) {
```

Comment:
I feel that what I did could be sub-optimal. Please let me know if this could be improved